src/pages/startGuest.tsx

```
@@ -75,6 +85,13 @@ const StartGuest = () => {
 
     return (
         <ResponsiveContainer>
+            <meta name="og:title" content={title} />
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)